src/views/Events/index.jsx

```diff
@@ -0,0 +1,207 @@
+import { Box, Container, makeStyles, Typography } from '@material-ui/core'
+import moment from 'moment'
+import { Calendar, momentLocalizer } from 'react-big-calendar'
+import EventCard from '../../components/EventCard'
+import EventSpotlight from '../../components/EventSpotlight'
+import Layout from '../../components/Layout'
+
+import 'react-big-calendar/lib/css/react-big-calendar.css'
+// import './react-big-calendar.css'
+
+const useStyles = makeStyles((theme) => ({
+  root: {},
+  eventsList: {
+    display: 'flex',
+    gap: '2em',
+    flexWrap: 'wrap',
+    alignItems: 'stretch',
+    justifyContent: 'center',
+    flexDirection: 'row',
+    height: 'max-content',
+    [theme.breakpoints.down('xs')]: {
+      flexDirection: 'column',
+      gap: '1em',
+    },
+  },
+  spotlight: {
+    display: 'flex',
+    alignItems: 'stretch',
+    gap: '2em',
+    [theme.breakpoints.down('md')]: {
+      flexDirection: 'column',
+      alignItems: 'stretch',
+      gap: '1em',
+      marginBottom: '3em',
+    },
+
+    '& .calendar': {
+      flex: '1 1 50%',
+
+      '& .rbc-calendar': {
+        height: '140vw',
+        maxHeight: '40em',
+        '& .rbc-month-view, .rbc-time-view, .rbc-week-view': {
+          borderRadius: '1em',
+          overflow: 'hidden',
+          '& .rbc-event': {
+            backgroundColor: '#4B87B1',
+          },
+        },
+      },
+    },
+    '& .event': {
+      flex: '1 1 50%',
+      [theme.breakpoints.down('md')]: {
+        display: 'flex',
+        justifyContent: 'center',
+      },
+    },
+  },
+}))
+
+const localizer = momentLocalizer(moment)
+
+const events = [
```

Comment:
Oh for this, can we place it in a src/data/events.js file, so it is easy to maintain the records all in one place